RetrospectiveExtension.Frontend/config/.gitignore

```diff
@@ -0,0 +1,2 @@
+# ignore enviornment containing possibly sensitive information such as the backend/ instrumentation key
+environment.tsx
```

Comment:
Or this separate `.gitignore` - it makes more sense to me to just have the toplevel one and specify all the files that may have sensitive keys in them.